autoPyTorch/datasets/base_dataset.py

```diff
@@ -186,34 +175,24 @@ def __getitem__(self, index: int, train: bool = True) -> Tuple[np.ndarray, ...]:
             A transformed single point prediction
         """
 
-        if hasattr(self.train_tensors[0], 'loc'):
-            X = self.train_tensors[0].iloc[[index]]
-        else:
-            X = self.train_tensors[0][index]
+        X = self.train_tensors[0].iloc[[index]] if hasattr(self.train_tensors[0], 'loc') \
```

Comment:
is there a reason why you are changing these if else to be in a single line?

Reply:
That is just because it is shorter and I use inline if statement whenever I can shorten lines. I use them. In other words, this is my convention. For me, It does not make sense to make lines longer unless the error statements become complicated.

Reply:
okay

Reply:
I think it makes sense, but only if you do not have a backslash. There is even a PEP rule for that which we should probably enable...